lib/proxy/helpers.rb

```diff
@@ -107,4 +107,19 @@ def remote_fqdn(forward_verify=true)
     end
   end
 
+  def ssl_client_cert(request)
+    if request.env.key?('SSL_CLIENT_CERT')
+      request.env['SSL_CLIENT_CERT'].to_s
```

Comment:
When is an env var not a string?

Reply:
When it's variable ;) Not apologizing for my bad joke :P

Reply:
I later noticed that the `.to_s` was already there in the previous code as well.